server/src/common/model/schema/session/session.schema.ts

```diff
@@ -0,0 +1,37 @@
+import { ISession } from "shared"
+
+import config from "@/config"
+
+import { model, Schema } from "../../../mongodb"
+
+export const sessionSchema = new Schema<ISession>(
+  {
+    token: {
+      type: String,
+      description: "Token de la session",
+    },
+    updated_at: {
+      type: Date,
+      default: Date.now,
+      description: "Date de mise à jour en base de données",
+    },
+    created_at: {
+      type: Date,
+      default: Date.now,
+      description: "Date d'ajout en base de données",
+    },
+    expires_at: {
+      type: Date,
+      default: Date.now,
+      description: "Date d'expiration",
+    },
+  },
+  {
+    versionKey: false,
+  }
+)
+
+sessionSchema.index({ token: 1 })
+sessionSchema.index({ expires_at: 1 }, { expireAfterSeconds: config.auth.session.cookie.maxAge / 100 })
```

Comment:
Non ici, soit tu fais `expireAfterSeconds: 0` et tu set le `expires_at` en fonction (donc pas `Date.now`) soit tu fais 
```suggestion
sessionSchema.index({ updated_at: 1 }, { expireAfterSeconds: config.auth.session.cookie.maxAge / 100 })
```

--> Et tu remove le champs

Reply:
c'est set mais justement cest le default pour etre certains